src/main/webapp/style.css

```diff
@@ -10,4 +10,33 @@ html {
 
 body #map{
     height: 400px;
-}
\ No newline at end of file
+}
+.title {
+  font-family: 'Trebuchet MS', sans-serif;
+  text-align: center;
+  font-weight: 900;
+  font-size: 80px;
```

Comment:
nit: here and below, alphabetize the css attributes in each class

Reply:
oh shoot I thought I did haha, Thanks Julie !